docker/Dockerfile

```diff
@@ -25,4 +25,4 @@ ENV PYTHONPATH /app
 # CMD ["sleep", "infinity"]
 
 # Can be replaced with the following to run directly:
-CMD ["uv", "run", "python", "main.py", "--mcp-host", "0.0.0.0", "--mcp-port", "8081"]
```

Comment:
I think I would like to keep the listening in the docker file instead of in main.py. Reaons:
1. more flexible on how to use relik tools
   * run local through main.py
   * run as a seperate docker container that listenes on host/port for remote MCP usage

By keeping the listening in the docker command instead of in main.py both scenarios can be executed on.

Reply:
Thanks heaps for the feedback.
Addressed as suggested.